microprofile/fault-tolerance/src/test/java/io/helidon/microprofile/faulttolerance/BulkheadBean.java

```diff
@@ -17,6 +17,8 @@
 package io.helidon.microprofile.faulttolerance;
 
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Semaphore;
```

Comment:
Usual comment about probably you want a `Lock` not a `Semaphore`, but perhaps you had your reasons.

Reply:
Done